### DISCUSSION TITLE
Documentation for Javascript methods

### DISCUSSION THREAD

**Discussion by un-clouded** [victordiaz/PHONK#104] [Q&A]
Phonk is such a breath of fresh air.  Thank you for making it!

Where is the documentation for the Javascript methods kept?  Is it generated from the comments on the Java methods such as in https://github.com/victordiaz/PHONK/blob/master/phonk_apprunner/src/main/java/io/phonk/runner/apprunner/api/network/PMqtt.java?

**Comment by victordiaz** [comment]
Hi @un-clouded,

Thanks for the comments :)

Yes, the documentation is autogenerated from the Java method signatures and comments.

To be honest the code I wrote to generate the documentation broke sometime ago because of some methods deprecation in Java that I was using. I stopped paying attention to the docs in consequence 😬😅
Now that you ask I will take the time to update it and have better documentation around that.

Feel free to keep asking!

Cheers

**Reply by un-clouded** [reply]
I would be happy to write some documentation for methods if that helps at all?

**Reply by victordiaz** [reply]
totally! :tada: I'd be very happy! Feel free to do it.
I'll try to find sometime this week to fix the broken doc generation and document the process :)
